packages/antd/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rjsf/antd",
-  "version": "3.2.0",
+  "version": "3.2.1",
```

Comment:
Can you revert the version bump changes? We'll do that once we cut a new version later.